### ISSUE TITLE
Changes to an aria-live region are not announced

### ISSUE THREAD

**Issue by JamieMagee** [nvaccess/nvda#7771]
### Steps to reproduce:

1. Open Firefox
2. Visit https://codepen.io/anon/pen/bYYKPE
3. Click `Add Content` button

### Expected behavior:

I expect that NVDA announces the addition of the text `Hello, world!` to the `aria-live` region.

### Actual behavior:

Nothing is announced.

### System configuration:
NVDA version: 2017.3

NVDA Installed or portable:

NVDA Installed

Other information:

n/a

Windows version:
Windows 10 Version 1709 Build 16299.19

Name and version of other software in use when reproducing the issue:

Firefox Developer Edition 58.0b4 (Though it also reproduces on standard Firefox builds)

### Other questions:

Does the issue still occur after restarting your PC?

Yes

Have you tried any other versions of NVDA?

No

**Comment by PratikP1** [comment]
Testing with Firefox 56 and Nightly with NVDA version next-14617,ef52ddc4, the given example speaks as expected. “Hello World” is announced each time “add content” button is pressed.

Testing with Chrome Canary Version 64.0.3271.0 results in no speech when “Add content” button is pressed.

Testing with Edge on Windows 10 1709 build 17040 results in no speech when “add content” button is pressed.

Similarly, there is no speech in IE a11.

**Comment by Adriani90** [comment]
With NVDA 2017.3, Firefox 57 and Build 16299.64 works properly. However, Crome, IE and Edge do not read the live region. But it could be an issue regarding Java Plugin.
